File: waterdrop-spark-api/src/main/scala/io/github/interestinglab/waterdrop/spark/structuredstream/StructuredStreamingSink.scala
package io.github.interestinglab.waterdrop.spark.structuredstream

import io.github.interestinglab.waterdrop.spark.BaseSparkSink
import org.apache.spark.sql.streaming.DataStreamWriter
import org.apache.spark.sql.Row

trait StructuredStreamingSink extends BaseSparkSink[DataStreamWriter[Row]]{

  
}

File: seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/structuredstream/StructuredStreamingSink.java
package org.apache.seatunnel.spark.structuredstream;

import org.apache.seatunnel.spark.BaseSparkSink;

import org.apache.spark.sql.Row;
import org.apache.spark.sql.streaming.DataStreamWriter;

@SuppressWarnings("PMD.AbstractClassShouldStartWithAbstractNamingRule")
public abstract class StructuredStreamingSink extends BaseSparkSink<DataStreamWriter<Row>> {
}
